packages/config/src/config.js

```diff
@@ -91,7 +92,7 @@ module.exports = ({
                                  * Context class is equal to app name and that class ass added to root element via the chrome-render-loader.
                                  */
                                 if (relativePath.match(/^src/)) {
-                                    return `.${appName}{\n${content}\n}`;
+                                    return `.${appName}${sassPrefix ? `, .${sassPrefix}` : ''}{\n${content}\n}`;
```

Comment:
I though the `sassPrefix` will replace the `appName` instead of having both.

Reply:
Mhh, yeah, that can be replaced completely.